### ISSUE TITLE
2 color dash line

### ISSUE THREAD

**Issue by sensescape** [tangrams/tron-style#26]
Hi! @patriciogonzalezvivo 

Could we try a dash line with 2 colors? And the ability to set the background color too?

<img width="453" alt="screen shot 2016-08-02 at 9 08 17 am" src="https://cloud.githubusercontent.com/assets/466585/17336067/62a8d354-5891-11e6-9867-9f402d07bb15.png">

**Comment by patriciogonzalezvivo** [comment]
Sure! We should sit down together and plan this... as part of the optimisation we should transform as much geometry we can to shaders...

Could you prepare some snapshots of routes styles we need... and what properties should be configurable like:

```
background_color
dash_width:
dash_height:
dash_space:
```

**Comment by sensescape** [comment]
Thanks! Sure, to add to the above:

background_color
dash_width:
dash_height:
dash_space:
dash1_color:
dash2_color:

Not sure yet how I'll use it, still sketching, but yes let's optimize when finalized

Like the highways? Should we change that to shader form?

**Comment by patriciogonzalezvivo** [comment]
Yes, right now... the road, glow, outline, and dashes add one geometry each one... that produce over draw + different custom shaders... all that slows down things... the idea is once we have something we like to put it together into less layers... 

probably the roads will contain the dashes and traffic animations (that they already had)
and the outline will provide also the glow... so the size need to be changed (bigger)

**Comment by sensescape** [comment]
Ok, I think the highway is ready to be converted to shader form.

**Comment by patriciogonzalezvivo** [comment]
future: speaking w @bcamper I realize this as a shader is more expensive... tangram (as is) have great support for pattern textures on lines... we should use that

**Comment by nvkelso** [comment]
Can you create a new issue to track this, please?

**Comment by patriciogonzalezvivo** [comment]
is technically "not an issue". Just a note for the future (because is a 2.1 goal) that this should be address with the existing lines texture patterns.
